### DISCUSSION TITLE
GitHub Actions is currently disabled for my account

### DISCUSSION THREAD

**Discussion by nocrynotear** [community/community#138910] [Actions]
### Select Topic Area

Bug

### Body

want to enable GitHub actions on my account. But I am getting below error on the top.
GitHub Actions is currently disabled for your account. Please reach out to [GitHub Support](https://support.github.com/contact?tags=dotcom-actions) for assistance.

**Comment by davevad93** [comment]
Hi @nocrynotear , 

Just as the warning states, contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem with GitHub Actions.

**Reply by nocrynotear** [reply]
okay。thanks

**Comment by cirosantilli** [comment]
Related: https://github.com/orgs/community/discussions/129404

**Comment by black-man2233** [comment]
Could it be the rate limit
